src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -11,7 +11,10 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Test;
 
+use Symfony\Bundle\FrameworkBundle\Tests\Functional\Kernel\TestKernel;
+use Symfony\Bundle\FrameworkBundle\Tests\Functional\Kernel\TestKernelInterface;
```

Comment:
just wondering: are these two classes required for KernelTestCase to work correctly?
because then that'd need a fix: no classes from the "Tests" namespace can be required for making things in "Test" work.